lib/index.js

```diff
@@ -252,6 +252,27 @@ class VideoAsset extends BaseAsset {
 
     set_download_uri(value) { this._download_uri = value; }
 
+    // Set a wrapper article asset ID on this video. This way,
+    // consumers can know if this article is merely just wrapping
+    // another video asset and can go directly to the video if
+    // appropriate
+    set_wrapper(article) {
+        // The only valid wrapper for VideoAsset is GalleryVideoArticle
+        if (!(article instanceof GalleryVideoArticle)) {
+            throw new Error(`The only valid wrapper for VideoAsset is GalleryVideoArticle`);
+        }
```

Comment:
Thanks for adding this. Thinking again about this, we could instead have a `wrappable` property that is `false` by default and set to `true` in the `GalleryVideoArticle` subclass